Library/Homebrew/os/mac/xcode.rb

```diff
@@ -216,7 +216,7 @@ def latest_version
         # on the older supported platform for that Xcode release, i.e there's no
         # CLT package for 10.11 that contains the Clang version from Xcode 8.
         case MacOS.version
-        when "10.13" then "900.0.22.8"
+        when "10.13" then "900.0.26"
```

Comment:
Aye! Didn't see any changes to the behaviour we were discussing the other day, but RL has prevented me actually being able to look particularly hard 😓.